documentation/staging/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -166,8 +164,8 @@ The following parameters can be provided in the inputs file.
 | `adminServerName` | Name of the Administration Server. | `admin-server` |
 | `clusterName` | Name of the WebLogic cluster instance to generate for the domain. | `cluster-1` |
 | `configuredManagedServerCount` | Number of Managed Server instances to generate for the domain. | `5` |
-| `createDomainWdtModel` | WDT model YAML file that the create domain script uses to create a WebLogic domain when using wdt mode. This value is ignored when WLST script is used in creating a WebLogic domain. | `wdt/wdt_model_dynamic.yaml` |
-| `createDomainWlstScript` | WLST script that the create domain script uses to create a WebLogic domain when using wlst mode. This value is ignored when WDT is used in creating a WebLogic domain. | `wlst/create-wls-domain.py` |
+| `createDomainWdtModel` | WDT model YAML file that the create domain script uses to create a WebLogic domain when using wdt `mode`. This value is ignored when WLST script is used in creating a WebLogic domain. | `wdt/wdt_model_dynamic.yaml` |
+| `createDomainWlstScript` | WLST script that the create domain script uses to create a WebLogic domain when using wlst `mode`. This value is ignored when WDT is used in creating a WebLogic domain. | `wlst/create-wls-domain.py` |
```

Comment:
How about: This value is ignored when `mode` is set to `wdt` (which is the default `mode`).